glean-core/src/ping/mod.rs

```diff
@@ -243,14 +245,23 @@ impl PingMaker {
         let temp_ping_path = temp_dir.join(doc_id);
         let ping_path = pings_dir.join(doc_id);
 
+        log::debug!("Storing ping '{}' at '{}'", doc_id, ping_path.display());
+
         {
             let mut file = File::create(&temp_ping_path)?;
             file.write_all(url_path.as_bytes())?;
             file.write_all(b"\n")?;
             file.write_all(::serde_json::to_string(ping_content)?.as_bytes())?;
         }
 
-        std::fs::rename(temp_ping_path, ping_path)?;
+        if let Err(e) = std::fs::rename(&temp_ping_path, &ping_path) {
```

Comment:
I see what you mean now. I think it should be ok to have this, it will save us some headache when debugging any problem.